include/pcp/common/axis_aligned_bounding_box.hpp

```diff
@@ -135,7 +169,7 @@ kd_bounding_box(ForwardIter begin, ForwardIter end, CoordinateMap const& coordin
     for (auto i = 0u; i < aabb.min.size(); ++i)
     {
         aabb.min[i] = std::numeric_limits<scalar_type>::max();
-        aabb.max[i] = std::numeric_limits<scalar_type>::min();
+        aabb.max[i] = std::numeric_limits<scalar_type>::lowest();
```

Comment:
Don't forget to also fix this same problem in the 3d axis aligned bounding box class!